mlflow/xgboost.py

```diff
@@ -130,7 +138,22 @@ def save_model(
                           model. The given example will be converted to a Pandas DataFrame and then
                           serialized to json using the Pandas split-oriented format. Bytes are
                           base64-encoded.
+    :param pip_requirements: Overwrites pip requirements of the conda environment this model should
+        be run in. Accepts either a string path to a pip requirements file on the local filesystem
+        or an iterable of pip requirement strings (e.g. :code:`["pandas", "-r requirements.txt"]`).
+    :param additional_pip_requirements: Works similarly to ``pip_requirements`` but specifies
+        additional pip requirements.
```

Comment:
Can we clarify that this adds pip requirements to a default set of requirements generated automatically by the model flavor based on the user's current software environment?

Reply:
Got it!